install.sh

```diff
@@ -268,6 +268,16 @@ rosdep install -r --from-paths src --ignore-src # Be careful not to install libo
 sudo apt purge ros-melodic-libg2o
 catkin_make
 
+# udev rules of opencm setting
+echo -e "\e[33m#####################################"
+echo -e "\e[33m# udev rules for opencm             #"
+echo -e "\e[33m#####################################"
+
+echo "ATTRS{idVendor}==\"fff1\" ATTRS{idProduct}==\"ff48\", ENV{ID_MM_DEVICE_IGNORE}=\"1\", MODE:=\"0666\", SYMLINK+=\"ultrasonics\"" >> 99-opencm-cdc.rules
+sudo mv 99-opencm-cdc.rules /etc/udev/rules.d/
+sudo udevadm control --reload-rules
+sudo udevadm trigger
```

Comment:
I'm not so familiar with udev rurle, but can `udevadm` command be used before [line 314](https://github.com/SeaosRobotics/setupJetsonTX2-Xavier/blob/da3c9129470298b2824cc5218bb9570633e75b32/install.sh#L314) `sudo apt-get install libudev-dev`?
FYI, k2k/install.sh just do cp, chown, and chmod (https://github.com/SeaosRobotics/k2k/blob/50a1cbf65f8e12003875fb64c2a9e5ebdb5cc96b/service/install.sh#L15)